src/toffy/watcher_callbacks.py

```diff
@@ -358,16 +363,6 @@ def build_callbacks(
         run_callbacks + intermediate_callbacks if intermediate_callbacks else run_callbacks[:]
     )
 
-    for run_cb in callbacks_with_prereq:
-        argnames = inspect.getfullargspec(getattr(RunCallbacks, run_cb))[0]
-        argnames = [argname for argname in argnames if argname != "self"]
-
-        misc_utils.verify_in_list(required_arguments=argnames, passed_arguments=list(kwargs.keys()))
-
-    callbacks_with_prereq = (
-        run_callbacks + intermediate_callbacks if intermediate_callbacks else run_callbacks[:]
-    )
-
```

Comment:
Can you explain why this is this getting removed?

Reply:
This particular section was duplicated, so it should be removed.